packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -105,6 +105,7 @@ export function generateRuntimeOptions(
 		enableRuntimeIdCompressor: ["on", undefined, "delayed"],
 		enableGroupedBatching: [true, false],
 		explicitSchemaControl: [true, false],
+		noLocalIdsForAttachmentBlobs: [undefined],
```

Comment:
why it's not [true, false]?
I.e. the assumption with all such properties is that transition from old to new format is not happening atomic, i.e. there will be (new) clients in a session that use either of the options.

Reply:
Changed it to [true, false] after updating logic to disable GC when `noLocalIdsForAttachmentBlobs` is turned on.